test/server/listener_manager_impl_test.cc

```diff
@@ -123,6 +127,13 @@ class ListenerManagerImplWithRealFiltersTest : public ListenerManagerImplTest {
   }
 };
 
+class MockAddressResolver : public Network::Address::Resolver {
```

Comment:
test nit: I would move this into the common mocks as it is generally useful

Reply:
thanks for the review. done.